docs/concepts/services-networking/service.md

```diff
@@ -203,6 +203,35 @@ having working [readiness probes](/docs/tasks/configure-pod-container/configure-
 
 ![Services overview diagram for iptables proxy](/images/docs/services-iptables-overview.svg)
 
+### Proxy-mode: ipvs
+
+In this mode, kube-proxy watches Kubernetes `services` and `endpoints`,
+call `netlink` interface create ipvs `virtual server` and `real server` accordingly.
```

Comment:
and it creates IPVS "virtual server" and "real server" accordingly.

- a reader doesn't care the fact that `netlink` interface exists and gets called;
- `virtual server` and `real server` are not supposed to be verbatim.